assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -121,6 +121,7 @@ const CardPreviewContainer = ({
   story,
   pageSize,
   children,
+  onClickCard = () => {},
```

Comment:
if anyone has a better name to describe a click on the entire card I'm not tied to this. When I set up the card UI I had thought we would have targeted actions within the card but not on the card itself in addition, thus the 'targetAction.centerAction/topAction/bottomAction' structure.

Reply:
Do we have access to `targetAction` here?  You could potentially have `targetAction.cardClick`

Reply:
the `templateActions` are divvied up in the `CardPreviewContainer` for where they are the action as well as the label, in keeping with that I opted to rename `onClickCard` because it's a terrible name to `containerAction` which suits this much better I think. Thanks for helping me think differently about this.